backend/cmake/MacOSXFrameworkInfo.plist.in

```diff
@@ -17,7 +17,7 @@
     <key>CFBundleVersion</key>
     <string>1</string>
     <key>CFBundleShortVersionString</key>
-    <string>${VERSION_STR}</string>
+    <string>{{ DRONECODE_SDK_VERSION }}</string>
```

Comment:
So what was wrong with `${VERSION_STR}`? Was that not actually working?

Reply:
Ooooh, I thought it was something related to Xcode (e.g. that Xcode would interprete somehow). Or maybe I was assuming that it was magically replaced by cmake in your previous implementation with the version file, I don't know.

Anyway, since I sed in the whole file, I'm a bit nervous about replacing `${VERSION_STR}`, which seems like it may be used by something else 😅. But I can change it back if you want.

Reply:
It's replaced by cmake automatically as far as I know.

Reply:
But not anymore, right? Or do we have git logic inside cmake for setting the version?

Reply:
Should still work:
https://github.com/Dronecode/DronecodeSDK/blob/20cd93a218b552aaab60d86a7848dd78f7407add/CMakeLists.txt#L65-L70
and:
https://github.com/Dronecode/DronecodeSDK/blob/20cd93a218b552aaab60d86a7848dd78f7407add/backend/src/CMakeLists.txt#L63

Reply:
My mistake, I had misread the error message:

```ERROR ITMS-90060: This bundle is invalid. The value for key CFBundleShortVersionString 'v0.12.0-2-gd842bd2b' in the Info.plist file must be a period-separated list of at most three non-negative integers.```

The ${VERSION} logic works, but the value we assign to it is not valid for an Info.plist.

I can get my packaging script to fix the Info.plist when invalid (`v0.12.0-2-gd842bd2b` -> `0.12.0`), and the deployment script to reuse that one.

Reply:
Well, if you deploy without a tag that's what you get, yes. So, now you could strip the git hash, etc. for this case but I would do it directly in cmake or similar to the script which gets the version initially.

Reply:
But if I change `$VERSION_STR`, it will affect the debug info when running the SDK, and you will not know anymore if somebody is running from a release or from a "dirty" hash.

Reply:
Or maybe I should just set it to `0.0.0` if it is invalid in the Info.plist. And fail to deploy it.

Reply:
No. I will leave it incorrect in the Info.plist, and the deployment script will just fail in this case.